tests/main/snap-advise-command/task.yaml

```diff
@@ -27,7 +27,14 @@ execute: |
        fi
        sleep 1
     done
-    stat /var/cache/snapd/commands.db
+    if ! stat /var/cache/snapd/commands.db; then
+        # workaround for misbehaving store
+        if journalctl -u snapd | MATCH "429 Too Many Requests"; then
```

Comment:
please use the tool "journal-state get-log" instead which takes into consideration the cursor for the last test

Reply:
Done.